specs/cpts/Receptionist.cpt

```diff
@@ -145,3 +141,46 @@
 * Click Accept & Proceed button for aadhaar verification
 * Enter OTP "000000" for Aadhaar verification
 
+# Create a new patient with random details and random ABHA ID with authentication type as Demographics
```

Comment:
This needs to be deleted. duplicate of # Create a new patient with random details and random ABHA ID with authentication type as <AuthType>